ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/cygwin/tasks/main.yml

```diff
@@ -15,35 +15,21 @@
   register: cygwin_installed
   tags: cygwin
 
-# For details about the cache, see: https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/1210#issuecomment-679076545
-- name: Retrieve cygwin cache from ci.adoptopenjdk.net
+- name: Retrieve Cygwin setup
   win_get_url:
-    url: https://ci.adoptopenjdk.net/userContent/winansible/cygwin64.tar.gz
-    dest: C:/temp/cygwin64.tar.gz
-    checksum: 8f8b1d7d00233e384e725390c4018941fe7125290b94ecc0eeb3cf6684b4ce23
+    url: https://cygwin.com/setup-x86_64.exe
+    dest: C:\temp\cygwin.exe
+    checksum: 4dd4d4531e8e63ade849daaaf587ba1c1430368701772c8ee42a27f4e8c373e4
```

Comment:
do we want to checksum this? it's a dynamic link that changes each time they bump the version so will require regular playbook updates

Reply:
I was somewhat concerned about that - the alternative is to not have the checksum, or to do what we're looking at with the Visual Studio installers

Reply:
I would remove it for now.

Reply:
Fixed :+1: